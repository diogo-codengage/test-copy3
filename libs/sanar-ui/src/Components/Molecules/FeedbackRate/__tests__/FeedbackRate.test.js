
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESFeedbackRate from '../'

it('renders correctly', () => {
    const component = (
        <ESFeedbackRate>
            <h1>ESFeedbackRate</h1>
        </ESFeedbackRate>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
