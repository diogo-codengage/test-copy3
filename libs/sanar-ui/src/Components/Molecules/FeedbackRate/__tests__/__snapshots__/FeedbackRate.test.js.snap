// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-feedback-rate"
  style={
    Object {
      "display": "flex",
    }
  }
>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "alignItems": "center",
        "cursor": "pointer",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "opacity": "1",
      }
    }
  >
    <img
      alt={1}
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "padding": "12px",
        }
      }
    />
    <span>
      Péssima
    </span>
  </div>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "alignItems": "center",
        "cursor": "pointer",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "opacity": "1",
      }
    }
  >
    <img
      alt={2}
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "padding": "12px",
        }
      }
    />
    <span>
      Ruin
    </span>
  </div>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "alignItems": "center",
        "cursor": "pointer",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "opacity": "1",
      }
    }
  >
    <img
      alt={3}
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "padding": "12px",
        }
      }
    />
    <span>
      Regular
    </span>
  </div>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "alignItems": "center",
        "cursor": "pointer",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "opacity": "1",
      }
    }
  >
    <img
      alt={4}
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "padding": "12px",
        }
      }
    />
    <span>
      Boa
    </span>
  </div>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "alignItems": "center",
        "cursor": "pointer",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "opacity": "1",
      }
    }
  >
    <img
      alt={5}
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "padding": "12px",
        }
      }
    />
    <span>
      Incrível
    </span>
  </div>
</div>
`;
